### DISCUSSION TITLE
Is there an good client o viewer in android?

### DISCUSSION THREAD

**Discussion by albelixpd** [zim-desktop-wiki/zim-desktop-wiki#2277] [General]
I use zim desktop to organize personal informations.
actually I sync the data with my android device and read data by a text editor (DroidVim), but i'd like use a wysiwyg editor. 
is it exists?

Thanks

**Comment by dehesselle** [comment]
Have you seen this https://github.com/zim-desktop-wiki/zim-desktop-wiki/discussions/1737#discussioncomment-1613477 ?

**Reply by albelixpd** [reply]
Thank you! It seems interesting, I did not understand that it also manages the Zim-Wiki format.
About the MD format.... unfortunately I don't appreciate the MD format, because in a Wysiwyg editor the inline images are not supported.

For now I will try Markor.
Thanks again.
